code/game/mecha/mecha.dm

```diff
@@ -132,6 +132,7 @@
 	var/defence_mode = FALSE
 	var/defence_mode_deflect_chance = 15
 	var/leg_overload_mode = FALSE
+	var/coral_leg_overload_mode = FALSE
```

Comment:
This variable can be moved to the action itself, it isn't used anywhere outside of it.

Reply:
it doesn't like it when I move it to the action unfortunately.

Reply:
@SapphicOverload